quake1_example/class/func_train.gd

```diff
@@ -68,6 +53,29 @@ func _start(c : Curve3D) :
 	player_end = false
 	_play_snd('plats/train1.wav')
 
+
 func _f() :
 	player_end = true
 	_play_snd('plats/train2.wav')
+
```

Comment:
This is mostly untouched, just formatting, moved down sound function to be consistent with other func scripts